sdk/cognitivelanguage/azure-ai-language-questionanswering/swagger/README.md

```diff
@@ -20,25 +20,58 @@ autorest
 ### Settings
 
 ```yaml
-input-file: https://raw.githubusercontent.com/Azure/azure-rest-api-specs/34a2c0723155d134311419fd997925ce96b85bec/specification/cognitiveservices/data-plane/Language/stable/2021-10-01/questionanswering.json
-output-folder: ../azure/ai/language/questionanswering
 namespace: azure.ai.language.questionanswering
 package-name: azure-ai-language-questionanswering
 license-header: MICROSOFT_MIT_NO_VERSION
 clear-output-folder: true
 no-namespace-folders: true
 python: true
-title: QuestionAnsweringClient
 version-tolerant: true
-models-mode: msrest
-package-version: 1.1.0b1
+package-version: 1.1.0b3
 add-credential: true
 credential-default-policy-type: AzureKeyCredentialPolicy
 credential-key-header-name: Ocp-Apim-Subscription-Key
 black: true
 ```
 
-### Rename "QuestionAnsweringKnowledgeBase_Query" -> "GetAnswers"
+## Batch Execution
+
+```yaml
+batch:
+  - tag: release_runtime_1_1_preview
+  - tag: release_authoring_1_1_preview
+```
+
+
+## Runtime
+
+These settings apply only when `--tag=release_runtime_1_1_preview` is specified on the command line.
+
+```yaml $(tag) == 'release_runtime_1_1_preview'
+input-file: https://raw.githubusercontent.com/Azure/azure-rest-api-specs/34a2c0723155d134311419fd997925ce96b85bec/specification/cognitiveservices/data-plane/Language/stable/2021-10-01/questionanswering.json
+output-folder: ../azure/ai/language/questionanswering
+models-mode: msrest
```

Comment:
@kristapratico I believe we want to eventually add new DPG models for authoring. Are we able to move the existing QnA models over to DPG as well / are there plans to?

Reply:
I have not heard plans. This release is intended to stabilize the authoring client. Are DPG models soon to be GA?

Reply:
Discussed offline. Authoring doesn't use models yet (so adding DPG models in the future is not breaking). Runtime already GA'd with msrest models.